generators/contract/templates/v2/private/typescript/src/my-contract.ts

```diff
@@ -74,14 +85,16 @@ export class <%= assetPascalCase %>Contract extends Contract {
         if (!exists) {
             throw new Error(`The asset <%= assetSpaceSeparator %> ${<%= assetCamelCase %>Id} does not exist`);
         }
+
+        const myCollectionName: string = await getCollectionName(ctx);
```

Comment:
Might seem a bit of a strange request, but can we change all instances of `myCollectionName` across the all of the languages to be just `collectionName` or `collection` - just to keep it simpler!

Reply:
Yeah I'll change it to `collectionName`